pull request for cute endpoints(⁠｡⁠･⁠ω⁠･⁠｡⁠)⁠ﾉ⁠♡

zxhiend opened the pull request:
can you pwease review my code and test it in ur device to see if it's working locally nyan~? awlsoo, don't forget to install faker module w/ `npm install @faker-js/faker --save-dev` nyannn<3. thank youww

zxhiend commented:
Added another cute endpoint "/asciiArt" that returned cuteeee ascii art & emojis nyan:3333

BeSalt1080 commented:
@zxhiend faiz-kun~! Could you pretty please use LF endings instead of those pesky CRLF ones? 🥺👉👈 The linter-chan gets all flustered and tsundere when it sees them~! 💻💕 Arigatouuuu~! 💖😳✨

zxhiend commented:
sowwyyy, but this issue will be pending until  my coffee hangout is over tonight nyann(⁠๑⁠•⁠﹏⁠•⁠) but for the forgorten /asciiArt schema is naw uploaded!(⁠ ⁠╹⁠▽⁠╹⁠ ⁠)

zxhiend commented:
Added another kawaii endpoint "/funfact" that returned cool random funfact:333.

Alsooo, I added usefuwl feature in all endpoints I made that are delete data that you deemed unworthy(๑>◡<๑): 
`DELETE /api/<endpoint_name>/<id>`

and reset the database in case the endpoint returned buggy n rude data(  ≧ᗜ≦):
`POST /api/<endpoint_name>/reset`

zxhiend commented:
security update will be implemented soon becaws my fren is afraid of bad heckers(,,>﹏<,,)

zxhiend commented:
Shinichann~ I added `/images` endpoint like you requested<3

I make sure the endpoint has cool delet and kawaii reset features like other endpointt:3 but heckers still could abuse the features(˚ ˃̣̣̥⌓˂̣̣̥ ), i'll leave the security patch to youuヾ( ˃ᴗ˂ )◞ • *✰

BeSalt1080 reviewed (approved):
I tested the images, the credit card, fun facts, and those super cute ascii arts endpoints. it all worked really well (≧◡≦)✧

Please wait just a lil' longer for the authentication features, okay~? (｡•́︿•̀｡) They're coming soooon~! (⁄ ⁄>⁄ ▽ ⁄<⁄)

zxhiend commented:
thank you for merginggg<33
